abixen-platform-web-client/src/main/web/application/modules/user/scss/styles.scss

```diff
@@ -14,24 +14,14 @@
   }
 }
 
-.user-details {
-  .avatar {
-    img {
-      height: 150px;
-      border: 1px solid black;
-      padding: 10px;
-      @include border-radius(5px);
-    }
-    a {
-      float: right;
-      margin-top: 10px;
-    }
-  }
-  .user-id {
-    margin-top: 108px;
-  }
+.avatar {
```

Comment:
After modification avatar is displaying as block. For this reason is not need to add 108 pixel of margin-top. User id field will be displayed under the image